devops/README.md

```diff
@@ -0,0 +1,70 @@
+# DevOps challenge
+
+This challenge is used by Didomi for evaluating candidates for DevOps positions.
+
+The challenge is a chance for engineers at Didomi to see how you deploy architecture and organize a project.
+
+## Deliverables
+
+The expected deliverable is a a project that includes the following:
+
+- All the resources/configuration to be deployed
+- Documentation and comments where appropriate
+
+## Technical stack
+
+For the purpose of this challenge you will use the following:
+
+- [terraform](https://www.terraform.io/) for deploying the resources/infrastructure
+- [serverless](https://www.serverless.com/) for deploying the lambda
+
+It's up to you to decide how much infrastructure you would like to deploy via serverless.
+
+## Expectations
+
+Your code will be reviewed by multiple engineers at Didomi and will serve as the base for a discussion in interviews.  
+We want to see how you approach working on a complete project and strongly recommend that you work on this challenge alone. We expect the code to be professionally structured, commented, and documented.
+
+If anything is unclear, feel free to ask any questions that might help you understand the specifications or requirements better.
+
+## Delivery
+
+Your application can be sent to us as a GitHub repository or as a compressed archive containing all the deliverables.
+
+# The challenge
+
+The Preference Management Platform (PMP) team has decided on their architecture. They want to create an API using the following technologies.
+
+- an [API Gateway](https://aws.amazon.com/api-gateway/)
+- a [Postgresql Database](https://aws.amazon.com/rds/)
+- a single [lambda](https://aws.amazon.com/lambda/) deployed with [serverless](https://www.serverless.com/). The lambda needs to make requests to a 3rd party public API.
+
+![Architecture](/devops/architecture/PMP%20Architecture.png)
+
+## Product specification
+
+To keep the challenge as brief as possible we advise you focus strictly on defining resources. To focus efforts consider the following:
+
+- We do not expect this stack to be deployed
+- We do not expect any programmatic logic in the Lambda, please use a placeholder (eg. a mock handler that always returns 200)
+- We do not expect a CI/CD process
+- Connecting the lambda to the RDS is not in the scope of this challenge
```

Comment:
I think what should be in scope is network and database configuration to allow that but agreed the actual code connecting is useless here